examples/5-eks-cluster-with-windows-support/main.tf

```diff
@@ -108,7 +107,12 @@ module "aws_eks_accelerator" {
   create_eks         = true
   kubernetes_version = local.kubernetes_version
 
+  # Ensure AWS VPC CNI is used
+  enable_vpc_cni_addon  = true
+  vpc_cni_addon_version = "v1.10.1-eksbuild.1"
+
   # MANAGED NODE GROUPS
+  enable_managed_nodegroups = true # default false
```

Comment:
This won't be needed anymore.This can cause error

Reply:
Removed it.